aea/crypto/base.py

```diff
@@ -179,27 +221,24 @@ def get_balance(self, address: Address) -> Optional[int]:
         """
 
     @abstractmethod
-    def transfer(
```

Comment:
The transfer method does not fit the message based approach and is therefore removed. the same behaviour can be obtained by a) requesting a tx; b) signing it; c) sending it